src/moscot/backends/ott/_utils.py

```diff
@@ -1,18 +1,20 @@
-from typing import Any, Optional
+import inspect
+from typing import Any, Callable, Dict, Optional
 
+import jax.numpy as jnp
 from ott.geometry.pointcloud import PointCloud
 from ott.tools.sinkhorn_divergence import sinkhorn_divergence
-import jax.numpy as jnp
 
-from moscot._types import ArrayLike, ScaleCost_t
 from moscot._logging import logger
+from moscot._types import ArrayLike, ScaleCost_t
 
 
 def _compute_sinkhorn_divergence(
     point_cloud_1: ArrayLike,
     point_cloud_2: ArrayLike,
     a: Optional[ArrayLike] = None,
     b: Optional[ArrayLike] = None,
+    # TODO(MUCDK): why is this so high?
```

Comment:
Nice spot. When I wrote this function I used it for comparing distances between distributions of non-scaled gene expression data, which sometimes covers a big range, as it is not normalized (and we don't want to scale the cost because then different scales of differences are not recoverable any more).

I just want to make sure that it converges. Still, I think it is unreasonably high. Maybe set it to 1e-1?

Reply:
Ok, will use `1e-1`.